CHANGELOG.md

```diff
@@ -437,3 +437,4 @@
 [@skryukov]: https://github.com/skryukov
 [@johnsyweb]: https://github.com/johnsyweb
 [@theunraveler]: https://github.com/theunraveler
+[@supersam654]: https://github.com/supersam654
```

Comment:
This is not needed to my understanding, the script will add this when the new version is released.